### PR TITLE
[StructuralMechanicsApplication] fix adjoint tests warnings

### PR DESCRIPTION
**Pull request by armingeiser**

### CODE REVIEW

File: applications/StructuralMechanicsApplication/tests/response_function_tests/rectangular_plate_parameters.json [hunk 1]
```diff
@@ -32,9 +32,6 @@
         "max_iteration"                      : 10,
         "problem_domain_sub_model_part_list" : ["Parts_AREAS"],
         "processes_sub_model_part_list"      : ["DISPLACEMENT_DIR_BC","PointLoad3D_NODES"],
-        "linear_solver_settings"       : {
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/tests/response_function_tests/rectangular_plate_parameters.json]: is this test constructed through the test-factory?

Comment (armingeiser) [reply applications/StructuralMechanicsApplication/tests/response_function_tests/rectangular_plate_parameters.json]: No, fixed it

**Comment by philbucher** [comment]
looks good to me

**Comment by MFusseder** [comment]
Looks reasonably to me.

**Comment by armingeiser** [comment]
=) can one of you approve it?

**Review by MFusseder — APPROVED** [review]

**Comment by loumalouomega** [comment]
@armingeiser  do you still need the branch?

**Pull request merged**